.github/workflows/rust.yml

```diff
@@ -13,7 +13,8 @@ jobs:
       matrix:
         build:
           - wasm32
-          - macos
+          # TODO: Too slow; see https://github.com/actions/runner-images/issues/1336
```

Comment:
Could we time a full rebuild? If it's only a few hours it might still make sense to run on master (just not blocking merges).

Reply:
The problem is that it's very unpredictable, but I've only seen it running a bit above 30 minutes. Shouldn't take hours.

Done.